modules/UI/videolayout/RemoteVideo.js

```diff
@@ -576,6 +576,9 @@ RemoteVideo.prototype.addRemoteStreamElement = function (stream) {
 
     if (!isVideo) {
         this._audioStreamElement = streamElement;
+
+        // for the volume slider
```

Comment:
Could you expand on this comment? "For the volume slider" means re-render the remote video menu to ensure the menu is displayed because volume should now be adjustable?